brainiak/hyperparamopt/mcmc.py

```diff
@@ -0,0 +1,133 @@
+#  Copyright 2016 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""Metropolis-Hasting Random number generator
+
+This implementation provides random samples from a user-given
```

Comment:
Or maybe http://dan.iel.fm/emcee/current/ . It's OK if neither of these does what you need, you can just say that (and why) in the comments so people know there's a reason for having an MCMC module in brainiak rather than using an existing library.